kungpao/display.py

```diff
@@ -114,7 +114,10 @@ def display_single(img,
                    color_bar_width='75%',
                    color_bar_height='5%',
                    color_bar_fontsize=18,
-                   color_bar_color='w'):
+                   color_bar_color='w',
+                   add_text = None,
+                   text_fontsize = 30,
```

Comment:
Very small format comment.  It is usually good practice to follow the python style guide PEP 8: https://www.python.org/dev/peps/pep-0008/ 

For parameters, there should be no space around `=`: `add_text=True`.